src/Config.php

```diff
@@ -1284,6 +1287,21 @@ function formatGlobalLayout(layout) {
             }
             echo "</td></tr>";
 
+            echo "<tr class='tab_bg_2'><td><label for='timeline_relative_date$rand'>" . __('Timeline date display') .
+            "</label></td><td>";
+            if (!$userpref || Session::haveRight('ticket', Ticket::OWN)) {
+                Dropdown::showFromArray('timeline_relative_date', [
+                    self::TIMELINE_RELATIVE_DATE => __('Relative'),
+                    self::TIMELINE_ABSOLUTE_DATE => __('Absolute'),
```

Comment:
```suggestion
                    self::TIMELINE_ABSOLUTE_DATE => __('Precise'),
```

The constant may be renamed in consequence